lib/Settings/Setting_wrapper.dart

```diff
@@ -0,0 +1,128 @@
+import 'package:cboard_mobile/Settings/ThemeManager.dart';
+import 'package:cboard_mobile/Settings/routes.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'routes.dart';
+
+class Setting_wrapper extends StatelessWidget {
+  // ThemeNotifier theme = new ThemeNotifier();
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      theme: ThemeData(
+        primaryColor: Color(0xFF23036A),
+        accentColor: Color(0xFFE3CAFF),
+        visualDensity: VisualDensity.adaptivePlatformDensity,
+        scaffoldBackgroundColor: Colors.white,
+      ),
+      home: Scaffold(
+        backgroundColor: Color(0xffE5E5E5),
+        appBar: AppBar(
+          centerTitle: true,
+          title: Text('Settings'),
+        ),
+        body: MyStatefulWidget(),
+      ),
+      routes: getRoute(),
+    );
+  }
+}
+
+class MyStatefulWidget extends StatefulWidget {
+  MyStatefulWidget({Key key}) : super(key: key);
+  @override
+  _MyStatefulWidget createState() => _MyStatefulWidget();
+}
+
+class _MyStatefulWidget extends State<MyStatefulWidget> {
```

Comment:
I see that you created a `StatefulWidget` here - if the `Settings_wrapper` widget does not contain any persistent state, can we change this to a `StatelessWidget`?

Reply:
It makes sense. I changed it. Thanks for pointing it out